### PR TITLE
Fixed DO group permissions to allow for writing the connection string

### PR DESCRIPTION
**Pull request by nihemstr**
Added the code in our postisnt script to add adu to the do group so we have permissions to write the connection_string to DO in order to fix an issue with the Nested Edge scenario.

### CODE REVIEW

**Review by Nox-MSFT — CHANGES_REQUESTED** [review]
:clock1:

File: packages/debian/postinst [hunk 1]
```diff
@@ -152,6 +152,17 @@ case "$1" in
             systemctl restart "$ms_doclient_lite_service"
         fi
 
+        # Note: We must add the 'adu' user to the 'do' group so that we can set the connection_string for
+        # the nested edge scenario. 
+        echo "Add the 'adu' user to the 'do' group"
+        if getent passwd 'adu' > /dev/null; then
```
Comment (JeffMill) [inline packages/debian/postinst:158]: we dont own the DO group.  Why are "we" doing this?

Comment (JeffMill) [reply packages/debian/postinst:158]: related to above where we are adding do user to adu group.

**Review by JeffMill — CHANGES_REQUESTED** [review]
:clock1:

File: packages/debian/postinst [hunk 2]
```diff
@@ -143,9 +143,11 @@ case "$1" in
         # which is one of the dependencies declared in adu-agent control file.
         # We are assuming that both DO user and group currently exist at this point.
         # Add 'do' user to 'adu' group to allow DO to write to ADU download sandbox.
-        echo "Add the 'do' user to the 'adu' group."
+        echo "Add the 'do' user to the 'adu' group and 'adu' to the 'do' group"
         if getent passwd 'do' > /dev/null; then
             usermod -aG 'adu' 'do'
+            # Note: We must add the 'adu' user to the 'do' group so that we can set the connection_string for DO
+            usermod -aG 'do' 'adu'
```
Comment (JeffMill) [inline packages/debian/postinst:150]: >adu [](http://example.com/codeflow?start=30&length=3)

`$adu_user` ?

**Review by JeffMill — APPROVED** [review]
:shipit:

**Review by Nox-MSFT — APPROVED** [review]
:shipit:

**Pull request merged**